src/docs/spell.ok

```diff
@@ -187,6 +187,7 @@ autogen
 automake
 basecfg
 basho
+behaviour
```

Comment:
I think MongoDB and WT use American spelling (behavior and optimized) in public documents.  I would leave out behaviour and optimised from this list and change the text.

Reply:
That's true, thanks! [9b21f18](https://github.com/wiredtiger/wiredtiger/pull/8367/commits/9b21f186bb973ba5c89e96925bf8becfecf64b56)